user_guides/websocket/src/main/jbake/content/attributes.conf

```diff
@@ -20,7 +20,7 @@
 //
 // The environment variable used to specify the home directory
 // for the technology.  Used in config.inc.
-:TechnologyHomeEnv: TS_HOME
+:TechnologyHomeEnv: WEBSOCKET_HOME
```

Comment:
All references are for TS_HOME, should we change to WEBSOCKET_HOME?

Reply:
`TS_HOME` is named explicitly in the text, TechnologyHomeEnv is the VI home, the home of the vendor implementation. TS_HOME is the TCK Suite home. See other technology TechnologyHomeEnv settings, non seems to be set to TS_HOME.

Reply:
Well, Concurrency, SAAJ, and Servlet have it set to TS_HOME, which is not correct, possibly. The others to VI home, or JAVAEE_HOME.

Reply:
Note that `TechnologyHomeEnv` is used just in `config.inc`.